File: res/users/random_user/problems/1_sumNums/main.py
class Solution():
    def sum_numbers(self, num1, num2):
        print("Hello world")
        return num1 + num2

args = [[1, 3], [3, 5], [7, 9]]
expected = [4, 8, 16]
f = open('user.out', 'a')
try:
	for i in range(len(args)):
		value = Solution().sum_numbers(args[i][0], args[i][1])
		if not isinstance(value, int): raise TypeError(f'{str(value)} is not valid value for the expected return type int')
		if value != expected[i]: raise ValueError(f'Expected value of {str(expected[i])}, but got {str(value)}')
		f.write(f'{str(value)}\n')
except Exception as e:
	raise e

File: res/users/random_user/problems/1_sumNums/main.cs
using System;
using System.IO;
public class Program {

public class Solution {
    public int sumNumbers(int num1, int num2) {
        Console.WriteLine("Hello world!");
        return num1 + num2;
    }
}

	public static void Main() {
		int[,] args = new int[,] {{1, 3}, {3, 5}, {7, 9}};
		int[] expected = {4, 8, 16};
		Solution solution = new Solution();
		try {
			using (StreamWriter writer = new StreamWriter("user.out"))
				for (int i = 0; i < args.GetLength(0); i++) {
					int value = solution.sumNumbers(args[i,0], args[i,1]);
					if (value != expected[i])
						throw new ArgumentException(String.Format("Expected value of {{0}}, but got {{1}}", expected[i], value));
					writer.Write(value + "\n");
				}
		} catch (Exception e) {
			throw e;
		}
	}
}
